server/seeders/007.exhibition.seed.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.bulkInsert('Exhibitions', [
+      {
+        date: Date.parse('2017-10-12 12:00:00.0003'),
+        place: 'Lviv',
+        firstPlaceWinnerId: 1,
+        secondPlaceWinnerId: 2,
+        thirdPlaceWinnerId: 3,
+        name: 'Charly',
+        createdAt: Date.now(),
+        updatedAt: Date.now()
+      },
+      {
+        date: Date.parse('2017-10-12 13:30:00.0003'),
+        place: 'Lviv',
+        firstPlaceWinnerId: 2,
+        secondPlaceWinnerId: 3,
+        thirdPlaceWinnerId: 4,
+        name: 'Mikky',
+        createdAt: Date.now(),
+        updatedAt: Date.now()
+      },
+      {
+        date: Date.parse('2017-10-19 12:00:00.0003'),
+        place: 'Kyiv',
+        firstPlaceWinnerId: 2,
+        secondPlaceWinnerId: 3,
+        thirdPlaceWinnerId: 4,
```

Comment:
When all seed is running, it will be the mistake, because we don't have pigeon in our DB with id="4".